### ISSUE TITLE
Please use unix style line ending

### ISSUE THREAD

**Issue by jingqi** [tumashu/posframe#16]
I'm installing `posframe` in background with [`emacs-async`](https://github.com/jwiegley/emacs-async), an error raises:
```
Failed to install `posframe':
	posframe.el:0:0: error: error: (Local variables entry is missing the suffix)
```
I guess the line end style (windows CRLF in posframe.el) cause this problem, and Emacs got an unexpected character '\r' at the end of the first line.
Please change to Unix line end style to fix this.

**Comment by tumashu** [comment]
posframe is using utf-8-unix

**Comment by jingqi** [comment]
Thanks for your response.
Maybe it's some default coding things in forked process, I'll dig that. 
This issue could be closed 😄

**Comment by jingqi** [comment]
I thank I've found the problem:
In file `posframe.el`, there is a local variable:
```
;; Local Variables:
;; coding: utf-8-unix
;; End:
```
which is usually `utf-8` in some other Emacs packages.

In Windows OS system, When posframe is downloaded in process created by emacs-async, somehow the file content was converted to '\r\n' line end style(maybe by download module of windows Emacs), but buffer coding system is force specified `utf-8-unix` by local variable and Emacs thanks the extra '\r' is wrong..

I've tried to set default buffer coding system to `utf-8` / `utf-8-unix` / `utf-8-dos` in subprocess, but it dosen't work. Maybe we should change `coding: utf-8-unix` to `coding: utf-8`?
